tree-sitter-ruby/.ghci

```diff
@@ -0,0 +1 @@
+../.ghci
```

Comment:
This is a symlink, yeah? And not e.g. a regular file with these contents?

Reply:
Yup. symlink.